src/widget/wsearchlineedit.cpp

```diff
@@ -465,27 +472,32 @@ void WSearchLineEdit::slotTriggerSearch() {
 
 /// saves the current query as selection
 void WSearchLineEdit::slotSaveSearch() {
+    m_saveTimer.stop();
+    QString cText = currentText();
```

Comment:
Here we should apply `trim()` to avoid storing empty searches (see below, the check for `isEmpty())

Reply:
:+1: